Tasks/JenkinsDownloadArtifacts/tslint.json

```diff
@@ -0,0 +1,60 @@
+{
+  "rules": {
```

Comment:
Changes local to the repo (as in this [pull request](https://github.com/Microsoft/vsts-tasks/pull/4698)) are what causes the tslint.json to be used. Giving each task/team the ability to control their linting rules allows them to manage their code how they want. Lacking a global way to do linting across this repository, that pull request is the suggestion I made. If you want to go another direction, that's fine. Feel free to reject that PR.